core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -633,18 +633,20 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options) {
     try {
       // Allocate from given location.
       dirView = mAllocator.allocateBlockWithView(sessionId, options.getSize(),
-          options.getLocation(), allocatorView);
-
+          options.getLocation(), allocatorView, false);
       if (dirView != null) {
         return dirView;
       }
 
       if (options.isForceLocation()) {
         if (options.isEvictionAllowed()) {
+          LOG.debug("Free space for block expansion: freeing {} on {}. ",
+                  options.getSize(), options.getLocation());
```

Comment:
Will this be extremely verbose?

Reply:
I think this should be okay. At least for my case, when we turn on debug log for `TieredBlockStore` there are something going wrong with tiered allocation, and space freeing is definitely a very critical part that we want to look into. Plus we don't free space on every buffer flush (every `freeSpace()` cleans up at least one block), so I think it's okay.